03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,71 @@
+                            need to be real for in order for it to be cathartic?
+                        </td>
+                        <td>
+                            This can be disconcerting, to have one’s feelings ranked or measurably ignored this way. As odd as it may be for your feelings to be sitting alongside someone’s retweet of a Laverne Cox tweet and an advertisement for British Airways, it feels even odder
+                            for those tweets to garner more engagement. But attention metrics don’t merely throw all utterances into competition. For an emotional tweet, the metrics can make sympathy and relatability appear essentially the same. Transforming
+                            sadness into textual performance and refining it to make it reverberate with the broadest possible audience is a new way to cleanse oneself of the pain by releasing it into the world, lessening one’s own feelings of isolation
+                            while securing a potentially viral consolation. It is digitally native catharsis. Melissa Broder’s @SoSadToday exemplifies this type of cathartic tweeting — anguish, sadness, and fear encapsulated into single tweets that make
+                            it easy to retweet and express such uncomfortable feelings as this. The humor centers the audience rather than the feeling, even as the audience’s response shows that the feeling has become expressible, tangible, understandable.
+                            The “realness” of the emotions expressed are confirmed not by their intrinsic intensity or supposed spontaneity but in the reaction they get. Making an audience laugh at your feelings may thereby be among the best ways to master
+                            them.
+                        </td>
+                    </tr>
+                </table>
+            </article>
+        </section>
+        <aside>
+            <img src="http://www.emcmarketing.com/images/ad_rates/samples/bcha-buyers-guide/1-2-vertical.jpg" />
+        </aside>
+        <footer>
+            <table>
+                <tr>
+                    <td>409-748-0857</td>
+                </tr>
+                <tr>
+                    <td><a href="mailto:dailyplanetcontact@dailyplanet.com">Email Us</a></td>
+                </tr>
+                <tr>
+                    <td>
+                        <a href="https://www.facebook.com/"><img src="http://findicons.com/files/icons/816/icons_web_2_0/128/facebook.png"></a>
+                        <a href="https://www.twitter.com/"><img src="http://findicons.com/files/icons/2137/twitter/128/twitter_11.png"></a>
+                        <a href="https://www.instagram.com/"><img src="http://findicons.com/files/icons/2745/sugar/128/instagram.png" ?></a>
+                    </td>
+                </tr>
+            </table>
+    </main>
+    </footer>
+</body>
+
+</html>

```

Comment:
Fantastic! The email link is clever. Next time, link to real social media accounts, if you want.